### DISCUSSION TITLE
post/cellular-automata-tutorial

### DISCUSSION THREAD

**Discussion by giscus** [foopod/blog6#10] [General]
# post/cellular-automata-tutorial

https://jonoshields.com/post/cellular-automata-tutorial

**Comment by ClementineAccount** [comment]
I like your blog! I found this post from https://www.reddit.com/r/godot/comments/15o37km/i_made_conways_game_of_life_tutorial_in_comments/. Was researching different ways of custom rendering for Godot (yours uses the sprite but its still interesting to read). 

I do think that your posts on your blog would benefit from having a date/time stamp. It helps with seeing how long the gap between posts were and when the original post was written (in case details have changed over time).
